lib/verify.js

```diff
@@ -332,6 +332,71 @@ async function VerifyProofs(proofData, options) {
   return verifyResults
 }
 
+// define a function for verifying a set of proofs sharing a common nodeUri
+async function verifySetAsync(proofDataSet) {
+  let firstAttempt = true
+  return retry(
+    async () => {
+      let targetURI = firstAttempt ? proofDataSet.nodeUri : null
+      firstAttempt = false
+      return chp.verifyProofs(proofDataSet.proofs, targetURI)
+    },
+    {
+      retries: 2, // The maximum amount of times to retry the operation. Default is 10
+      factor: 1, // The exponential factor to use. Default is 2
+      minTimeout: 10, // The number of milliseconds before starting the first retry. Default is 1000
+      maxTimeout: 10
+    }
+  )
+}
+
+async function verifyBtcProof(proof, client) {
+  if (!client) throw new Error('Must pass a client to verifyBtcProof')
+  try {
+    // get header from bitcoin node
+    let header = await client.getBlock(parseInt(proof.anchor_id, 10))
+
+    // check if there was a header returned and it matches the expected value in the proof
+    let verified = header && header.merkleRoot === proof.expected_value
+    if (!verified) {
```

Comment:
@jasonbukowski if you're interested, here's the check I'm putting in for now to see if a proof verification failed because of a "network mismatch". It's a little inexact, but in the narrow scope of my testing where I have some proofs anchored using the legacy system and some on the new, it seems to work well enough. 

Here's an example of the output:
<img width="1440" alt="Screen Shot 2019-05-28 at 5 21 50 PM" src="https://user-images.githubusercontent.com/4344978/58516309-1f6a7680-816d-11e9-8816-380455221fad.png">